compiler/loco/src/IR/Graph.test.cpp

```diff
@@ -79,7 +79,11 @@ TEST(GraphTest, create_and_destroy_node)
   auto pull = g->nodes()->create<loco::Pull>();
 
   ASSERT_NO_THROW(g->nodes()->destroy(pull));
-  ASSERT_THROW(g->nodes()->destroy(pull), std::invalid_argument);
```

Comment:
``nodes()`` returns NodeContext

https://github.com/Samsung/ONE/blob/cd3ae1e46dd947e320d4bced110effa4879b1dbe/compiler/loco/include/loco/IR/Graph.h#L218

NodeContext is an alias of NodePool:
https://github.com/Samsung/ONE/blob/cd3ae1e46dd947e320d4bced110effa4879b1dbe/compiler/loco/include/loco/IR/Graph.h#L170

``NodePool::destroy`` internally invokes ``ObjectPool::erase``:
https://github.com/Samsung/ONE/blob/cd3ae1e46dd947e320d4bced110effa4879b1dbe/compiler/loco/include/loco/IR/NodePool.h#L44-L50

``ObjectPool::erase`` internally checks the existence of node..
https://github.com/Samsung/ONE/blob/cd3ae1e46dd947e320d4bced110effa4879b1dbe/compiler/loco/include/loco/ADT/ObjectPool.h#L52-L69